upup/models/cloudup/resources/addons/cluster-autoscaler.addons.k8s.io/k8s-1.15.yaml.template

```diff
@@ -258,6 +258,22 @@ spec:
     app.kubernetes.io/name: "cluster-autoscaler"
   type: "ClusterIP"
 ---
+{{- if (eq GetClusterAutoscalerExpander "priority") }}
```

Comment:
I am guessing this will break things for everyone already using priority if they are not also setting the priority values and letting kOps override the CM correctly. There need to be a migration path of some sort for this.

One way of doing it would be to require the value being set on all IGs with autoscaling enabled. That would however break things for those using regex, which means you may need an option for "bring your own CM".

Reply:
I'll take a look on way to not break things for users that already use `priority` expander with their own configMap and add the ability to use a custom configMap.
I'm thinking about:
1. adding a new field on cluster autoscaler which will take precedence on `autoscalePriority` field of instance group if defined. 
Like:
```yaml
clusterAutoscaler:
  priorityExpanderConfig: |-
    100:
    - foo.*
    200:
    - bar.foo.*
```
2. Adding a field on cluster autoscaler in order to allow self manage configMap for cluster autoscaler. 
Like:
```yaml
clusterAutoscaler:
  selfManagedPriorityExpanderConfig: true
```